src/values/UnDeserializableValue.js

```diff
@@ -52,18 +52,18 @@ var SELF = dv.UnDeserializableValue = util.inherit(
 	_targetType: null,
 
 	/**
-	 * @property {Error}
+	 * @property {string}
 	 * @private
 	 */
-	_unserializeError: null,
+	_deserializeError: null,
 
 	/**
 	 * @inheritdoc
 	 *
-	 * @return {string}
+	 * @return {int}
 	 */
 	getSortKey: function() {
-		return this.getReason().name;
+		return 0;
```

Comment:
Why not keep `return this.getReason();` or simply use `return this._deserializeError;`. It's guaranteed to be a string.